### ISSUE TITLE
 selector_separator_newline adds erroneous newline on @extend SCSS statements

### ISSUE THREAD

**Issue by TomAnthony** [beautify-web/js-beautify#1799]
# Description

The `selector_separator_newline` option, when set to `true`, is slightly buggy with SCSS `@extend` statements, adding a newline when it should not.

# Input
The code looked like this before beautification:

```scss
	a {
		@extend .color-black, .hover-far-red;
	}
```

# Expected Output
The code should have looked like this after beautification:

```scss
	a {
		@extend .color-black, .hover-far-red;
	}
```

# Actual Output
The  code actually looked like this after beautification:

```scss
	a {
		@extend .color-black,
		.hover-far-red;
	}
```

The added newline is erroneous.

It works correctly for `@import` statements with comma delimited entries, and does not add a new line.

# Steps to Reproduce

## Settings

Example:
```json
{
    "selector_separator_newline": false
}
```

**Comment by damienfa** [comment]
Same issue everywhere there is a "comma separated list" . 

Example with list variable definition in SCSS :

**Expected :** 
`$default-margin: (desktop: 24px, tablet: 16px, phone: 16px);`

**Actual :** 
```
$layout-grid-default-margin: (desktop: 24px,
tablet: 16px,
phone: 16px,
);
```

**Comment by mhnaeem** [comment]
@bitwiseman This was fixed in #1800
